conf/pacific/upgrades/5_0_upgrade_small-cluster.yaml

```diff
@@ -0,0 +1,39 @@
+globals:
```

Comment:
Due to pool creation issues with small configuration, small-cluster global conf was removed. we can use `conf/pacific/upgrades/upgrades.yaml` for 5.1 as well with NFS.

Reply:
This file may not be required, if we use `conf/pacific/upgrades/upgrades.yaml`.

Reply:
I think it would be nice if we can keep a conf handy in case if any execution to be done on an upgraded cluster and to have a minimal cluster with lesser amount of time and resources.

Reply:
Planning to remove this and keep only one small cluster conf